BookFinder/src/components/book.vue

```diff
@@ -0,0 +1,197 @@
+</a>         <v-list-tile-sub-title><span class='text--primary'>{{author.average_rating}}</span></v-list-tile-sub-title>
+                      </v-list-tile-content>
+                    </v-list-tile>
+                    </div>
+</span>        <span>
+             <v-subheader>Similar Books</v-subheader>
+
+                    <div v-for="(book, key) in result.similar_books" v-bind:key="key">
+                        <v-list-tile :key="book.title" avatar>
+                      <router-link :to="'/book/Goodreads/'+book.id">
+                         <v-list-tile-avatar>
+                        <img :src="book.small_image_url" >
+                      </v-list-tile-avatar>
+                      </router-link>
+                      <v-list-tile-content>
+                      <router-link :to="'/book/Goodreads/'+book.id">
+                        <v-list-tile-title v-html="book.title"></v-list-tile-title>
+                      </router-link>
+                        <v-list-tile-sub-title><span class='text--primary'>{{book.authors.author.name}}</span>&mdash; {{book.average_rating}}</v-list-tile-sub-title>
+                      </v-list-tile-content>
+                    </v-list-tile>
+                    </div>
+
+
+        </span>
+          </div>
+</v-card-title>
+<!-- @TODO FEATURES TO BE ADDED-->
+        <v-card-actions>
+          <v-btn flat color="orange">read</v-btn>
+          <v-btn flat color="orange">share</v-btn>
+</v-card-actions>
+                </div>
+                                    <div v-else-if="result.error">  
+                      <v-list-tile>
+                      <v-list-tile-content>
+                        <v-list-tile-title >No search results found in {{index}}.</v-list-tile-title>
+                      </v-list-tile-content>
+                    </v-list-tile>
+                      </div>
+                    <v-divider ></v-divider>
+
+              </v-card>
+            </v-flex>
+          </v-layout>
+        </v-container>
+
+      </v-content>
+    </v-app>
+
+</div>
+
+
+</template>
+ 
+<script>
+export default {
+    name : 'book',
+    data(){
+        return {
+            result: {},
+            id: this.$route.params.id,
+            src: this.$route.params.src
+        }
+    },
+    methods: {
+        getBook(src,id){
+            switch (src){
+            case 'Goodreads':
+              //  this.$set(this,'result',this.GetbookGoodread(this.id)); //   
+              this.result = this.GetbookGoodread(id);
+                break;
+            case 'Google':
+                this.result = this.GetbookGoogle(id);
+                break;
+            default: 
+                this.$router.push('/');
+           }
+
+
+        },
+        GetbookGoodread: function (id){
+            var ResultsObj = {};
+            this.$http.get('https://cors-anywhere.herokuapp.com/https://www.goodreads.com/book/show/'+id+'.xml?key=whM4DZOLcr7BsKwZs1Gclw')
```

Comment:
You have exposed your API key to the public. Make sure you remove it from here and also regenerate your key.

Reply:
I think we can make use of dotenv here https://www.npmjs.com/package/dotenv